scripts/testnet/local-osmosis-testnet-new.sh

```diff
@@ -1,6 +1,11 @@
 #!/bin/bash
+
 set -e
 
+SCRIPT_DIR=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &>/dev/null && pwd)
+BINS_DIR=$SCRIPT_DIR/bins
+PATH=$BINS_DIR:$PATH
```

Comment:
Should we export it? It doesn't seem to be used in this file

Reply:
Those variables are used to extend the `$PATH` variable, allowing the invocation of various binaries without specifying their path.